setup.py

```diff
@@ -104,6 +112,18 @@
     ],
     ext_modules=cythonize(
         module_list=[
+            setuptools.Extension(
```

Comment:
Is this necessary? Doesn't the `sources=['**/*.pyx']` underneath include everything?

Reply:
This is required as the files use numpy arrays. In this case, we must indicate the path to `numpy` which is done via the `get_include` method.

Alternatively, we can define the numpy path as a generic option, but I am not sure about the impact (if any) on cython code that does not use `numpy`.

Reply:
I simplified this in a61a9597c360687d911e4a90eaded8f8088b06cd by setting the path to numpy by default for all cython modules.